src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -116,6 +116,34 @@ class MessageHelper {
       const hmi_apis::Common_ServiceEvent::eType service_event,
       const hmi_apis::Common_ServiceUpdateReason::eType service_update_reason);
 
+  class ServiceStatusUpdateNotificationBuilder {
+   public:
+    typedef std::shared_ptr<ServiceStatusUpdateNotificationBuilder>
+        ServiceStatusUpdateNotificationBuilderSPtr;
+
+    static ServiceStatusUpdateNotificationBuilder
+    CreateServiceStatusUpdateNotificationBuilder(
+        hmi_apis::Common_ServiceType::eType service_type,
+        hmi_apis::Common_ServiceEvent::eType service_event);
+
+    ServiceStatusUpdateNotificationBuilder& add_app_id(const uint32_t app_id);
+
+    ServiceStatusUpdateNotificationBuilder& add_service_update_reason(
+        const hmi_apis::Common_ServiceUpdateReason::eType
+            service_update_reason);
+
+    operator smart_objects::SmartObjectSPtr();
```

Comment:
@mked-luxoft I would better create method, then casting operator, it would be more obvious. 
Also should it be const ?